kyo-core/shared/src/main/scala/kyo/streams.scala

```diff
@@ -143,7 +51,7 @@ object Stream:
                     def resume[T, U: Flat, S3](
                         command: Chunk[V],
                         k: T => U < (Streams[V] & S3)
-                    ) =
+                    )(using Tag[Streams[V]]) =
                         command.collectUnit(pf).andThen {
```

Comment:
Some of the methods that depend on a function and an internal state can't be cached because we'd need to allocate tuples on each effect handling, which is more expensive than allocating the handler.